test/unit/store/cart-store.js

```diff
@@ -21,7 +21,7 @@ describe('CartStore', () => {
         Dispatcher.handleAction('STUDENT_FOUND', {
             id: '123456',
             name: 'Pope Francis',
-            itemAddresses: []
+            items: []
```

Comment:
Did you run the UI to make sure this doesn't change anything.  I honestly have no clue if changing this would break things or not.